test/cpp/jit/test_gpu.cpp

```diff
@@ -3066,80 +3066,188 @@ void testGPU_FusionSimpleGemm() {
   }
 }
 
-// This test currently requires a combination of broadcast and reduction
-// operations and parellelization strategy that is currently not supported.
-// It is a goal to get this example working and this test is added so we
-// can continue working on getting this example fixed. Right now it
-// produces an incorrect result. Either we need to error coherently on the
-// optimization strategy we don't support and set this test to one we do support
-// or we need to get this schedule working correctly.
+// Softmax without input normalization. This is a simplified version
+// of testGPU_FusionSoftmaxNormalized.
 void testGPU_FusionSoftmax() {
   torch::jit::fuser::cuda::CudaKernel prog;
   Fusion& fusion = *prog.fusion_;
   FusionGuard fg(&fusion);
 
+  const int tidx = 32;
+  const int dimx = 32;
+  const int dimy = 16;
+  const int dimz = 130;
+
   // Set up your input tensor views
   TensorView* input_tv0 = makeDummyTensor(3);
```

Comment:
Since this is the basic form of softmax, can we use a simple 1-dimensional input?

Reply:
That's certainly possible. We just need to remove the `BID` parallelization and use just a single thread block. That said, another reason I added this was to highlight the extra step needed for the normalization, so I wanted also minimize the difference between this simplified version and the existing softmax test.

Reply:
no strong preference, but I think a bare bone version is more useful than trying to minimize the differences from the other softmax test.

Reply:
That also make sense, so I added 1D versions as well. We have 4 softmax tests: {1D or 3D} x {basic or input normalization}. Since softmax is an important target, I assume having 4 similar tests should be fine.

Reply:
thanks!